src/hotspot/cpu/riscv/riscv_v.ad

```diff
@@ -3663,6 +3663,38 @@ instruct vsignum_reg(vReg dst, vReg zero, vReg one, vRegMask_V0 v0) %{
   ins_pipe(pipe_slow);
 %}
 
+// ---------------- Round float/double Vector Operations ----------------
+
+instruct vround_f(vReg dst, vReg src, fRegF tmp, vRegMask_V0 v0) %{
+  match(Set dst (RoundVF src));
+  effect(TEMP_DEF dst, TEMP tmp);
```

Comment:
You might want to add `TEMP v0` in effect as v0 is clobbered in `java_round_float_v`. Similar for `vround_d`.

Reply:
Thanks for catching! Fixed.